lib/members_page.rb

```diff
@@ -0,0 +1,15 @@
+#!/bin/env ruby
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'scraped'
+
+class MembersPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::AbsoluteUrls
+
+  field :member_rows do
```

Comment:
The field name is part of the public interface of the class, so it's better to have it reflect what would make sense to someone consuming the data. For things like this it's probably better to have it appear in the resulting data as a list of 'members', rather than 'member_rows', which is largely an implementation detail. (i.e. if this ceased to be implemented with a table, we would want to change how the code _gets_ the information, but we shouldn't need to change how we _present_ the information)